test/units/galaxy/test_collection.py

```diff
@@ -40,9 +40,9 @@ def reset_cli_args():
 
 
 @pytest.fixture()
-def collection_input(tmp_path_factory):
+def collection_input(tmp_path_factory, path_sufix='test-ÅÑŚÌβŁÈ Collections Input'):
```

Comment:
Interesting.. I didn't know you can do that without making it a fixture.

Reply:
yup, as long as you can override the function parametrs

Reply:
TIL!

Reply:
Marking this thread as resolved.